interfaces/builtin/i2c_test.go

```diff
@@ -246,7 +246,7 @@ func (s *I2cInterfaceSuite) TestAppArmorSpecPath(c *C) {
 	c.Assert(spec.AddConnectedPlug(s.iface, s.testPlugPort1, s.testUDev1), IsNil)
 	c.Assert(spec.SecurityTags(), DeepEquals, []string{"snap.client-snap.app-accessing-1-port"})
 	c.Assert(spec.SnippetForTag("snap.client-snap.app-accessing-1-port"), testutil.Contains, `/dev/i2c-1 rw,`)
-	c.Assert(spec.SnippetForTag("snap.client-snap.app-accessing-1-port"), testutil.Contains, `/sys/devices/platform/{*,**.i2c}/i2c-1/** rw,`)
+	c.Assert(spec.SnippetForTag("snap.client-snap.app-accessing-1-port"), testutil.Contains, `/sys/devices/platform/{*,**.i2c}/i2c-1/** rw,  # Add any condensed parametric rules`)
```

Comment:
Can you add a test for 2 ports (which will show we end up with `i2c-{1,2}` instead of `{i2c-1,i2c-2}` when incorporating my suggestion, above)?

Reply:
Sure

Reply:
Done